inc/wp-components/classes/class-gutenberg-content.php

```diff
@@ -129,7 +129,7 @@ private function convert_block_to_component( $blocks, $current_block ) : array {
 			// Render block and clean up extraneous whitespace characters.
 			$content = render_block( $block );
 			$content = do_shortcode( $content );
-			$content = preg_replace( '/[\r\n\t\f\v]/', '', $content );
+			$content = preg_replace( '/[\n\r\t]/', ' ', $content );
```

Comment:
If this regex is from core, could the original be referenced? And is there supposed to be a space in the replacement?

Reply:
The original is in JS, and yeah, space is correct.